src/app/components/WorkoutCardBig/WorkoutCardBig.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import 'tailwindcss/tailwind.css';
+
+function WorkoutCardBig(): JSX.Element {
+  return (
+    <div className="w-80 rounded-3xl grid grid-cols-2 grid-rows-2">
```

Comment:
You could think about `article` instead of `div` here

Reply:
you're right! 

Reply:
I would like to create three different sizes of cards. So I plan to name the cards according to their size.

Reply:
Got it. Please take a look at my other comment :)

Reply:
You're definetly right. I will change it to an article. 👍🏼